modules/standard/Path.chpl

```diff
@@ -18,6 +18,28 @@
  */
 use Error;
 
+/* Returns a constant string representing the current directory according to
+   the operating system.  For Windows and POSIX, this is "."
+*/
+proc curDir(): string {
+  extern proc chpl_fs_curdir(): c_string;
```

Comment:
The c_string return type here should correspond to the declared return type in the supporting runtime code.